doc/whats_new/v0.24.rst

```diff
@@ -330,6 +330,12 @@ Changelog
   which clips the transformed values of test data to ``feature_range``.
   :pr:`17833` by :user:`Yashika Sharma <yashika51>`.
 
+- |Enhancement| Add value ``use_encoded_value`` for ``handle_unknown``
+  parameter and add ``unknown_value`` parameter to
```

Comment:
Since the whole `handle_unknown` parameter is new, maybe this could be something like

> Add a new `handle_unkown` parameter with a `use_encoded_value` option, along with a new `unknown_value` parameter to...